### DISCUSSION TITLE
How to Keep The Conversation Going?

### DISCUSSION THREAD

**Discussion by xbaha** [betalgo/openai#247] [Q&A]
I have a use case where I have a large set of categories, (around 1000 tokens), I want to send it one time, then ask ChatGPT to categorize company x, company y, etc.. from the categories set I have sent already.
The point is to reduce the cost of sending a fresh new prompt with the categories set every time.
How can I implement this?

**Comment by winzig** [comment]
I am not aware of any way to do this, currently. Even if you could, the cost would likely remain the same because the entire context needs to be presented to the model every time you want a response, and that large context is what drives the cost.

**Comment by Frogley** [comment]
It seems like you're doing some classification. Maybe consider using [Embeddings?](https://platform.openai.com/docs/guides/embeddings/what-are-embeddings)
